tools/emptyDropsWrapper/README.md

```diff
@@ -0,0 +1,15 @@
+# emptyDropsWrapper
+
+Reads in an rds file containing the count matrix in dgCMatrix format in gene x droplet orientation and produces a CSV file indicating cell filtering and other cell metadat as returned by the emptyDrops function.
+
+## Requirements
+
+This script requires the following R packages to be installed: 'optparse', 'DropletUtils'.
+
+DropletUtils are available from bioconductor and require R version 3.5
+
+https://bioconductor.org/packages/release/bioc/html/DropletUtils.html
+
+## Testing
+
+To run the tests go into the test directory and run test_emptyDropsWrapper.sh
```

Comment:
nit: run with `bash test_emptyDropsWrapper.sh` or `./test_emptyDropsWrapper.sh`

Reply:
Please mention what the path to the `test` directory is as well, e.g. 

"To run the tests go into the `/tools/emptyDropsWrapper/test/prepRDS.R` directory and run `bash test_emptyDropsWrapper.sh`"

Or consider using a code block for better readability: 
```
cd /tools/emptyDropsWrapper/test/prepRDS.R
bash test_emptyDropsWrapper.sh
```

Reply:
done